utilities/transactions/transaction_base.cc

```diff
@@ -486,6 +487,64 @@ Iterator* TransactionBaseImpl::GetIterator(const ReadOptions& read_options,
                                           &read_options);
 }
 
+template <typename IterType, typename ImplType, typename ErrorIteratorFuncType>
+std::unique_ptr<IterType> TransactionBaseImpl::NewMultiCfIterator(
```

Comment:
Forgot to mention - I'm planning to leverage the same logic for `CoalescingIterator` later, hence the template